### PR TITLE
[PTFE-215] 🐛 Workflow job error 500

### PR DESCRIPTION
**Pull request by gaspardmoindrot**

### CODE REVIEW

**Review by tcarmet — CHANGES_REQUESTED** [review]
Please add a testcase to ensure we won't encounter the issue in the future.

**Review by tcarmet — APPROVED** [review]
LGTM

Can you confirm that your test fails when the fix is not applied?

**Review by tcarmet — CHANGES_REQUESTED** [review]
unrelated commit in this PR.

File: tests/api/metrics/test_job.py [hunk 1]
```diff
@@ -101,3 +101,12 @@ def test_job_cost(client, workflow_job, headers):
     for line in metrics.text.split("\n"):
         if "github_actions_job_cost_count_total{" in line:
             assert "0.104" in line
+
+
+def test_skipped_job(client, workflow_job, headers):
```
Comment (tcarmet) [inline tests/api/metrics/test_job.py:106]: To reproduce the issue might need to add some config in the exporter

```suggestion
def test_skipped_job(override_job_config, client, workflow_job, headers):
```

File: tests/api/metrics/test_job.py [hunk 2]
```diff
@@ -101,3 +101,12 @@ def test_job_cost(client, workflow_job, headers):
...
+    workflow_job["workflow_job"]["runner_name"] = None
```
Comment (tcarmet) [inline tests/api/metrics/test_job.py:107]:
```suggestion
    # might be some empty labels that trigger it
    workflow_job["workflow_job"]["runner_name"] = None
    workflow_job["workflow_job"]["labels"] = []

```

**Comment by gaspardmoindrot** [comment]
Everything's good now!

**Review by tcarmet — APPROVED** [review]

**Pull request merged**